src/app/portlets/dot-edit-page/content/components/dot-edit-page-toolbar/dot-edit-page-toolbar.component.scss

```diff
@@ -1,16 +1,14 @@
 @import "variables";
 @import "mixins";
 
-$toolbar-height: 52px;
-
 :host {
     display: block;
     position: relative;
     z-index: 1;
 
     ::ng-deep {
```

Comment:
Why ng-deep if this is the component itself?

Reply:
actually this whole component will be removed, since it's not used anymore